### PR TITLE
Set the version of HEP_OSlibs correspondingly to the OS version

### PR DESCRIPTION
**Pull request by Pansanel**

### CODE REVIEW

File: personality/ui/rpms/config.pan [hunk 1]
```diff
@@ -7,7 +7,11 @@ unique template personality/ui/rpms/config;
 variable HEP_OSLIBS ?= true;
 "/software/packages" = {
     if (is_boolean(HEP_OSLIBS) && HEP_OSLIBS) {
-        SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        if (OS_VERSION_PARAMS['major'] == 'sl5') {
+            SELF[escape('HEP_OSlibs_SL5')] = nlist();
+        } else {
```
Comment (jrha) [inline personality/ui/rpms/config.pan]: Is `if sl5` ... `else sl6` safe and future proof? There is already a HEP_OSlibs_SL7.

You could do:
```pan
    "/software/packages" = pkg_repl(format('HEP_OSlibs_%s', to_uppercase(OS_VERSION_PARAMS['major'])));
```

Or perhaps:
```pan
variable hep_oslibs_mapping = dict(
    'sl5', 'HEP_OSlibs_SL5',
    'sl6', 'HEP_OSlibs_SL6',
    'sl7', 'HEP_OSlibs_SL7',
);

"/software/packages" = pkg_repl(hep_oslibs_mapping[OS_VERSION_PARAMS['major']]);
```

And maybe throw a nice error for other OS versions?

File: personality/wn/rpms/config.pan [hunk 2]
```diff
@@ -7,7 +7,11 @@ unique template personality/wn/rpms/config;
 variable HEP_OSLIBS ?= true;
 "/software/packages" = {
     if (is_boolean(HEP_OSLIBS) && HEP_OSLIBS) {
-        SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        if (OS_VERSION_PARAMS['major'] == 'sl5') {
+            SELF[escape('HEP_OSlibs_SL5')] = nlist();
+        } else { 
+            SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        };
```
Comment (jrha) [inline personality/wn/rpms/config.pan]: Could this not use the same code as above?
Or even be in a common template included by both `ui/rpms/config` and `wn/rpms/config`?

Comment (Pansanel) [inline personality/wn/rpms/config.pan]: I can create a new feature called 'wlcg', that could be include from wn and ui. What are you thinking about this?

Comment (jrha) [inline personality/wn/rpms/config.pan]: Sounds good if the name "wlcg" makes sense to everyone, the main goal is to reduce duplication across the templates.

`features/hep_oslibs/config` might be clearer unless you plan to put more general wlcg configuration in there as well.

**Comment by jrha** [comment]
I'll merge this an open an issue for 15.10 to move this to a common template.

**Pull request merged**